### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by lucabergamini**
Bump package version for kaggle competition release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,6 +38,9 @@ It consists of the following components:
 
 To read more about the dataset and how it was generated, read the [dataset whitepaper](https://arxiv.org/abs/2006.14480).
 
+**Note (08-24-20):** The new version of the dataset includes dynamic traffic lights support.
```
Comment (oliver-scheel) [inline README.md:41]: Minor: would put traffic light (singular)

File: README.md [hunk 2]
```diff
@@ -53,7 +57,7 @@ prediction-dataset/
   +- meta.json
 
 ```
-You may find other files and folder downloaded (mainly from `aerial_map`), but they are not required by L5Kit
+You may find other downloade files and folder (mainly from `aerial_map`), but they are not currently required by L5Kit
```
Comment (oliver-scheel) [inline README.md:60]: downloadeD ... folders

File: README.md [hunk 3]
```diff
@@ -53,7 +57,7 @@ prediction-dataset/
   +- meta.json
 
 ```
-You may find other files and folder downloaded (mainly from `aerial_map`), but they are not required by L5Kit
+You may find other downloaded files and folder (mainly from `aerial_map`), but they are not currently required by L5Kit
```
Comment (oliver-scheel) [inline README.md:60]: still: folderS (plural)

**Review by oliver-scheel — DISMISSED** [review]
Minor grammar, otherwise LGTM

**Review by stefaniespeichert — APPROVED** [review]
All LGTM could not find something wrong

**Pull request merged**